hawc/services/epa/dsstox.py

```diff
@@ -29,16 +30,15 @@ def create_from_dtxsid(cls, dtxsid: str) -> "DssSubstance":
         """
         if not re.compile(RE_DTXSID).fullmatch(dtxsid):
             raise ValueError(f"Invalid DTXSID: {dtxsid}")
-
         response = requests.get(
```

Comment:
throw a ValueError if settings.CCTE_API_KEY is None and settings.IS_TESTING is False